crates/workspace/src/workspace.rs

```diff
@@ -4122,7 +4119,7 @@ impl Workspace {
         if self._schedule_serialize.is_none() {
             self._schedule_serialize = Some(cx.spawn(|this, mut cx| async move {
                 cx.background_executor()
-                    .timer(Duration::from_millis(100))
+                    .timer(Duration::from_millis(300))
```

Comment:
I feel like 300ms is a little bit too long for this value. I'd like this time to be short enough that it's almost impossible for a human to move quickly enough to 'beat' the workspace serialization, so we don't ever lose a change due to bad timing. With 300ms, we're now waiting longer than the human reaction time.

Reply:
Sure, reverted that back.